### DISCUSSION TITLE
Doesnt work

### DISCUSSION THREAD

**Discussion by RayJooste** [rizwansoaib/whatsapp-monitor#220] [General]
Am i doing it wrong or has WhatsApp updated their privacy?

I've installed and run the extension exactly as described but cannot get it to work as i hoped it would.
I have two WhatsApp numbers so used this as my test.
I set the one phones WhatsApp privacy to "Nobody" on both Last seen and Online.
Logged into WhatsApp web using the other number and started to monitor the first phone, On the first phone (the one being monitored, I opened WhatsApp and then had a conversation with a friend of mine for a couple of minutes, making sure i went out of WhatsApp each time after responding to them (to create a log) and then reopening the app when they replied.
Downloaded to csv only to find the doc was empty.
I then closed closed WhatsApp web and started again, only this time i changed the privacy to Everyone in both instances on the first phone.
Redid the test and this time the csv had the details.
I tried this with other contacts as well and same result each time.

So it currently only gathers information on the online status as long as the target settings are not set to nobody.

Sorry for the long winded explanation but I just want to make sure its understood.
As far as I'm concerned, I'm either missing something or this exercise does not work on WhatsApp ver 2.24.17.79

responses and suggestions would be highly appreciated

**Comment by RayJooste** [comment]
please help I'm desperate

**Comment by rizwansoaib** [comment]
Yes you are right to track online activity user must allow online seen to
everyone or contacts if already chat happened then extension will work
